.circleci/config.yml

```diff
@@ -31,19 +31,19 @@ jobs:
       - run:
           name: Build
           command: make image
-      - run: echo "$GCR_JSON_KEY" | docker login -u _json_key --password-stdin us.gcr.io
+      - run: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
       - run:
-          name: Push image to GCR
+          name: Push image to Dockerhub
           command: |
-            docker tag codeclimate/codeclimate-sonar-php \
-              us.gcr.io/code-climate/codeclimate-sonar-php:b$CIRCLE_BUILD_NUM
-            docker push us.gcr.io/code-climate/codeclimate-sonar-php:b$CIRCLE_BUILD_NUM
+            make release RELEASE_TAG="b$CIRCLE_BUILD_NUM"
+            make release RELEASE_TAG="$(echo $CIRCLE_BRANCH | grep -oP 'channel/\K[\w\-]+')"
```

Comment:
@dantevvp  does this means we will be pushing images like `codeclimate-sonar-php:master` ? if so which service will be using it ? I'm asking because I don't remember having seen an image tag `:master` before.